app/models/restaurant.rb

```diff
@@ -4,7 +4,13 @@
 class Restaurant < ApplicationRecord
 
   belongs_to :cuisine
+  has_many :reviews
 
   # data validations:
   validates_presence_of :name
+
+  def update_rating
+    self.rating = reviews.sum(:rating) / reviews.length
```

Comment:
just FYI this code will fail if someone will create a review and than delete it since it will do
`x / 0`

Myabe add:
```ruby
self.rating = if reviews.length > 0
                reviews.sum(:rating) / reviews.length
              else
                 0
              end
```

Reply:
This is still relevant.